packages/workbox-webpack-plugin/index.js

```diff
@@ -62,6 +62,25 @@ class WorkboxBuildWebpackPlugin {
       config.globPatterns = ['**/*.{html,js,css}'];
     }
 
+    if (compilation.options.output.publicPath && !config.manifestTransforms) {
```

Comment:
yeah I have some thought about this and I might be just too conservative in my approach.
So, if our transformation started having some conflict with a user land's transformation function, then there is no way to switch off our logic. but as I said, I might be just over thinking this... Thoughts?